### DISCUSSION TITLE
how to clone repo

### DISCUSSION THREAD

**Discussion by renato67** [community/community#22288] [New to GitHub]
<p>Hi</p>
<p>does somebody can help me to clone a MKdocs repo on my area?</p>
<p>I would clone it, then install then access via web</p>
<p>Is it possible?</p>
<p>If yes can someone sudgest me documentation to do that?</p>
<p>TIA</p>
<p>Renato</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/renato67">@renato67</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>To clone a repository you can simply take the url give by the web ui:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d97a12beeb8b124d2943c584c600fea62e538cc0.png" data-download-href="/uploads/short-url/v1T9pooPl7fghTJcmE3v82NvdSw.png?dl=1" title="prova.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181070079-0682ef79-7bb4-4aeb-8051-fb012e9cd958.png" alt="prova.png" data-base62-sha1="v1T9pooPl7fghTJcmE3v82NvdSw" width="690" height="309" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d97a12beeb8b124d2943c584c600fea62e538cc0_2_690x309.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d97a12beeb8b124d2943c584c600fea62e538cc0_2_1035x463.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/d/d97a12beeb8b124d2943c584c600fea62e538cc0_2_1380x618.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d97a12beeb8b124d2943c584c600fea62e538cc0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">prova.png</span><span class="informations">1529×686 128 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Than git clone in local:</p>
<pre><code>git clone &lt;url&gt;
</code></pre>
<p>If the repository is this <a href="https://github.com/mkdocs/mkdocs" rel="nofollow noopener">https://github.com/mkdocs/mkdocs,</a> you can copy that git url and clone with this command:</p>
<pre><code>git clone https://github.com/mkdocs/mkdocs.git
</code></pre>
<p>To install you can follow this guide: <a href="https://www.mkdocs.org/#installation" rel="nofollow noopener">https://www.mkdocs.org/#installation</a></p>
<p>Hope to solve your problem :blush: </p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>
